components/massmint/OverviewTable.vue

```diff
@@ -5,21 +5,191 @@
+                </div>
+              </div>
+            </div>
+            <div class="column is-flex is-align-items-center">
+              <div class="height-50px is-flex is-align-items-center">
+                <div
+                  class="border is-size-7 is-justify-content-center py-1 my-2 is-flex is-align-items-center fixed-width"
+                  :class="statusClass(nft.status)">
+                  {{ nft.status }}
+                </div>
+              </div>
+            </div>
+            <div class="column is-flex is-align-items-center has-text-grey">
+              <NeoIcon
+                class="is-clickable"
+                icon="edit"
+                @click.native="openSideBarWith(nft)" />
+              <NeoIcon
+                class="is-clickable ml-3"
+                icon="trash"
+                @click.native="deleteNFT(nft)" />
+            </div>
+          </div>
+          <div ref="sentinel" />
+        </div>
       </template>
     </NeoCollapse>
   </div>
 </template>
 
 <script setup lang="ts">
-import { NeoCollapse } from '@kodadot1/brick'
+import { NeoCollapse, NeoIcon } from '@kodadot1/brick'
+import { useIntersectionObserver } from '@vueuse/core'
+import { NFT, NFTS, Status } from './types'
 
-withDefaults(
+const offset = ref(10)
+const sentinel = ref<HTMLDivElement | null>(null)
+const emit = defineEmits(['openSideBarWith', 'delete'])
+
+const props = withDefaults(
   defineProps<{
     disabled?: boolean
+    nfts?: NFTS
   }>(),
   {
     disabled: false,
+    nfts: undefined,
   }
 )
+
+const displayedNFTS = computed<NFT[]>(() =>
+  Object.values(props.nfts).slice(0, offset.value).map(addStatus)
+)
+
+const openSideBarWith = (nft: NFT) => {
+  emit('openSideBarWith', nft)
+}
+
+const deleteNFT = (nft: NFT) => {
+  emit('delete', nft)
+}
+
+const statusClass = (status?: Status) => {
+  const statusMap: { [status: string]: string } = {
+    Ok: 'k-greenaccent',
+    Incomplete: 'k-redaccent',
+    Description: 'k-yellow',
+  }
+
+  return status ? statusMap[status] : ''
+}
+
+const addStatus = (nft: NFT): NFT => {
+  let status: Status = 'Ok'
+  if (!nft.description) {
+    status = 'Description'
+  }
+  if (!nft.name) {
+    status = 'Incomplete'
+  }
+  return {
+    ...nft,
+    status,
+  }
+}
+
+const handleIntersection = (entries: IntersectionObserverEntry[]) => {
+  const target = entries[0]
+  if (target.isIntersecting) {
+    offset.value += 10
+  }
+}
+useIntersectionObserver(sentinel, handleIntersection, { threshold: 0.66 })
 </script>
+
+<style scoped lang="scss">
+@import '@/styles/abstracts/variables';
+
+.border-bottom {
+  @include ktheme() {
+    border-color: theme('k-shade');
+  }
+}
+
+.cover {
+  object-fit: cover;
+}
+
+.limit-height {
+  max-height: 30rem;
+  overflow-y: auto;
+}
+
+.fixed-width {
+  width: 80px;
+}
+
+.height-50px {
+  height: 50px;
+}
+
+//colums overrides
+.column {
+  padding: 0;
+  padding-right: 0.25rem;
+}
+
```

Comment:
this isn't right, maybe check [column gap](https://bulma.io/documentation/columns/gap)

Reply:
```is-gapless``` works
but is-<some-number> doesn't work (tried on chrome and firefox)

so i'd still need custom css

Reply:
Have you tried something like this `columns is-variable is-2`?

Reply:
you are right. Thank you!!

Didn't read the docs carefully enough :disappointed: